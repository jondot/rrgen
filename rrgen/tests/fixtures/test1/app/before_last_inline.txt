// Only the second struct should be injected

pub struct World1 {
  //fields
}

pub struct World2 {
  //fields
}
