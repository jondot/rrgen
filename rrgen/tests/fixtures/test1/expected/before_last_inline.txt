// Only the second struct should be injected

pub struct World1 {
  //fields
}

pub struct HelloWorld2 {
  //fields
}